libraries/AP_Param/AP_Param.cpp

```diff
@@ -1757,6 +1778,11 @@ AP_Param *AP_Param::next(ParamToken *token, enum ap_var_type *ptype, bool skip_d
         if (ptype != nullptr) {
             *ptype = AP_PARAM_FLOAT;
         }
+#if !APM_BUILD_TYPE(APM_BUILD_AP_Periph)
```

Comment:
`#if AP_PERIPH_DOWNLOAD_DEFAULTS_ENABLED` or something rather than referencing the build type everywhere.